### ISSUE TITLE
ncu causes NPM to create a local "etc" directory

### ISSUE THREAD

**Issue by rse** [tjunnone/npm-check-updates#193]
For unknown reasons, ncu seems to cause its backing NPM instance to create local "etc" directories:

```
$ ls -ld etc
ls: etc: No such file or directory
$ npm-check-updates  
[...]
$ ls -ld etc
drwxr-xr-x  2 rse  rse  2 Dec 25 12:00 etc
```
These "etc" directories are created on every ncu run again and again. I've traced it down to the NPM instance ncu uses: in "npm/lib/config/core.js" of NPM there is a function "afterUser" and at line 155 it performs the "mkdirp()" call which creates those directories. The reason seems to be that under ncu NPM's "prefix" config property is the local directory. I was still unable to figure out the root cause, but on regular NPM runs no such directory "etc" is created. But with ncu's NPM usage it is created. So, it looks like ncu's usage of NPM causes NPM to create those directories. I'm sure in older versions of ncu those directories were not created. Can someone check this in more detail and hopefully fix the problem?

**Comment by rse** [comment]
Ok, I've found the root cause in ncu:

```
Index: lib/package-managers/npm.js
--- lib/package-managers/npm.js.orig    2015-12-25 12:14:09.867877082 +0100
+++ lib/package-managers/npm.js 2015-12-25 12:14:17.925562836 +0100
@@ -52,8 +52,7 @@
 
         return npm.loadAsync({
             silent: true,
-            global: args.global,
-            prefix: args.prefix || ''
+            global: args.global
         })
         .then(function () {
             rawPromisify(npm.commands);
```
ncu as it looks does never provide a "prefix" in "args" and NPM dislikes an undefined value. Instead NPM wishes that the "prefix" option is set to a correct path or not present at all. As ncu does not use this option anyway, removing the option at all fixes the reported problem.

**Comment by cameronjroe** [comment]
+1 having the same issue. Not desirable.

**Comment by gsklee** [comment]
Same issue here.

**Comment by metaraine** [comment]
Thanks Ralf! I appreciate you digging into it. I published the fix in v2.5.6.
